### PR TITLE
Feature Forms

### PR DESCRIPTION
**Pull request by kelvinbush**
In this pull request I added forms to create new posts and comments, I also added a like button for the user to like a post.

### CODE REVIEW

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    else
+      render :new, status: :unprocessable_entity
+    end
+  end
+
+  private
+
+  def comment_params
+    params.permit(:text)
+  end
```
Comment (AnselemOdims) [inline app/controllers/comments_controller.rb:19]: - [ ] There should be a `new` method in the CommentsController class. Please kindly include a `new` method in this class.

File: config/routes.rb [hunk 2]
```diff
@@ -1,7 +1,13 @@
 Rails.application.routes.draw do
+  get 'likes/create'
+  get 'comment/new'
+  get 'comment/create'
   root to: 'users#index'
   resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show]
+    resources :posts, only: [:index, :show, :new, :create] do
+      resources :comments, only: [:index, :create]
```
Comment (AnselemOdims) [inline config/routes.rb:8]: - [ ] There should be a `new` route which is handled by the `new` method for the Comments. Please kindly include a `new` route for the Comments.

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @kelvinbush, 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please kindly include a view file for creating new comments. This view file can contain a form for creating new comments.
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: config/routes.rb (hunk 2 above)
Comment (kelvinbush) [reply config/routes.rb:8]: To add a comment navigate to the single post ( click on the post title) then there is a form where you can create a comment, the view file for the comment form is in the shared partials, as I am rendering it as a partial to the `post/show.html.erb`  view. The like button is also in this single post view.

**Review by AnselemOdims — APPROVED** [review]
### Hi @kelvinbush  🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**